<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portable Signal Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>signal-icon.jpg</normaloff>signal-icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 700 12pt &quot;Alef&quot;;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>901</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="waveform">
        <property name="text">
         <string>Waveform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSine">
        <property name="text">
         <string>Sine</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">waveformButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSquare">
        <property name="text">
         <string>Square</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">waveformButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTriangle">
        <property name="text">
         <string>Triangle</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">waveformButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSaw">
        <property name="text">
         <string>Sawtooth</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">waveformButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioPulse">
        <property name="text">
         <string>Pulse</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">waveformButtons</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="frequencyInput">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioHz">
        <property name="text">
         <string>Hz</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">amplitudeButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioKHz">
        <property name="text">
         <string>KHz</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">amplitudeButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMHz">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>MHz</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">amplitudeButtons</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amplitudeInput">
        <property name="text">
         <string>15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiomV">
        <property name="text">
         <string>mV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">frequencyButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioV">
        <property name="text">
         <string>V</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">frequencyButtons</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="offSetInput">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioomV">
        <property name="text">
         <string>mV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">offSetButtons</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiooV">
        <property name="text">
         <string>V</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">offSetButtons</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Duty Cycle (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dutyInput">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="generateSignalButton">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 181, 177, 255), stop:1 rgba(255, 255, 255, 255));
</string>
      </property>
      <property name="text">
       <string>Generate Signal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>901</width>
     <height>67</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 181, 177, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 20px;
height: 50px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Portable Signal Generator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="amplitudeButtons"/>
  <buttongroup name="offSetButtons"/>
  <buttongroup name="frequencyButtons"/>
  <buttongroup name="waveformButtons"/>
 </buttongroups>
</ui>
